backend/routes/PostManagement/PostDeletion.py

```
@@ -0,0 +1,28 @@
+from fastapi import APIRouter, Depends
+from sqlalchemy.orm import Session
+from api.v1.endpoints.Auth.auth import get_current_user
+from models.user import User
+from database.session import SessionLocal
+from sqlalchemy.orm import Session
```

Ruff (0.8.2)

Redefinition of unused `Session` from line 2

Remove definition: `Session`

(F811)